sound/soc/sof/sof-priv.h

```diff
@@ -411,10 +411,15 @@ struct snd_sof_dev {
 	struct snd_dma_buffer dmatb;
 	struct snd_dma_buffer dmatp;
 	int dma_trace_pages;
-	wait_queue_head_t trace_sleep;
 	u32 host_offset;
-	u32 dtrace_is_enabled;
-	u32 dtrace_error;
+	unsigned int dtrace_is_enabled:1; /* FW DMA trace is active */
+	unsigned int dtrace_error:1;      /* FW sent non-recoverable error */
```

Comment:
do we care about optimizing two bitfields?

Reply:
@plbossart This was more to set usage for future fields -- does not seem reasonable to keep adding u32 fields. But true, I was originally planning to add more fields, but now that I'm only adding one, it does indeed seem out of place. I'll drop.